### PR TITLE
Update new-to-ocaml.md

### PR DESCRIPTION
**Pull request by davesnx**
I would swap those paragraphs

### CODE REVIEW

File: docs/new-to-ocaml.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
-As a backend for the OCaml compiler, Melange shares many similarities with the
-OCaml language. Nevertheless, there are some notable differences between the
-two. This documentation aims to clarify these distinctions. For features that
-Melange inherits from OCaml, readers will be directed to the main OCaml
-documentation.
-
 If you are completely new to OCaml, it is recommended to familiarize yourself
 with the language first. [Learn OCaml](https://ocaml.org/docs) is a good
 starting point. but we recommend the following tutorials from the official OCaml
```
Comment (feihong) [inline docs/new-to-ocaml.md:5]:
```suggestion
starting point, and we also recommend the following tutorials from the official OCaml
```

File: docs/new-to-ocaml.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 website:
 
 - [A First Hour With OCaml](https://ocaml.org/docs/first-hour)
 - [OCaml Exercises](https://ocaml.org/exercises)
+
+As a backend for the OCaml compiler, Melange shares many similarities with the
```
Comment (feihong) [inline docs/new-to-ocaml.md:11]: Not related to this PR, but do most devs know what "backend of compiler" means?

**Review by feihong — APPROVED** [review]

File: docs/new-to-ocaml.md (hunk 2 above)
Comment (davesnx) [reply docs/new-to-ocaml.md:11]: True, worth explaining a bit here better

**Comment by feihong** [comment]
@davesnx Any reason to not merge this?

**Pull request merged**